### PR TITLE
VS1-Initial Slice

### PR DESCRIPTION
**Pull request by Michelbaartman**
https://trello.com/c/KNALZKBp/2-vs1-initial-slice

### CODE REVIEW

File: src/Command.hpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (wvanbreukelen) [inline src/Command.hpp]: As specified by the [definition of done](https://github.com/R2D2-2018/R2D2-2018/wiki/Definition-of-Done), code should be documentated. Unfortunately, in the (abstract) classes you have included within your pull request, this is not the case. Please document the future function of your class methods as specified on the [wiki](https://github.com/R2D2-2018/R2D2-2018/wiki/Code-standard).

File: src/main.cpp [hunk 2]
```diff
@@ -1,10 +1,14 @@
+#include "IO_Interface.hpp"
```
Comment (wvanbreukelen) [inline src/main.cpp:1]: Please write the filenames in the naming format specified on the [wiki](https://github.com/R2D2-2018/R2D2-2018/wiki/Code-standard).

File: src/IO_Interface.hpp [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+#ifndef IO_INTERFACE_HPP
+#define IO_INTERFACE_HPP
+
+#include "Command.hpp"
+
+class IOInterfaceListener;
+
+class IOInterface {
+  public:
```
Comment (wvanbreukelen) [inline src/IO_Interface.hpp]: Be consistent in your spacing. *For each indent 4 spaces will be used.*, [wiki](https://github.com/R2D2-2018/R2D2-2018/wiki/Code-standard#indentation)

**Review by wvanbreukelen — CHANGES_REQUESTED** [review]
Please update the code according to the placed comments throughout the code. Have a look at the code guidelines placed on the wiki.

**Comment by elsavira** [comment]
Requested changes have been made.

**Review by wvanbreukelen — APPROVED** [review]

**Pull request merged**